iam/api-client/src/test/java/iam/snippets/AccessTests.java

```diff
@@ -74,16 +93,16 @@ public void tearDown() {
   }
 
   @Test
-  public void testGetPolicy() {
-    GetPolicy.getPolicy("projects/" + PROJECT_ID);
+  public void testGetPolicy() throws IOException {
+    GetPolicy.getPolicy(PROJECT_ID, SERVICE_ACCOUNT);
     String got = bout.toString();
     assertThat(got, containsString("Policy retrieved: "));
```

Comment:
Instead of stdout, can we get the returned policy from `GetPolicy.getPolicy` and assert against it?
Same question for other testcases as well.